modules/spring-config/src/main/java/org/mule/runtime/config/internal/dsl/model/extension/xml/MacroExpansionModuleModel.java

```diff
@@ -297,6 +299,8 @@ private ComponentModel createModuleOperationChain(ComponentModel operationRefMod
     processorChainBuilder.addParameter("moduleOperation", operationModel.getName(), false);
     Map<String, String> propertiesMap = extractProperties(configRefName);
     Map<String, String> parametersMap = extractParameters(operationRefModel, operationModel.getAllParameterModels());
+    parametersMap.forEach((k, v) -> processorChainBuilder.addParameter(k, v, false));
```

Comment:
is this still needed?

Reply:
yes, this adds the parameters that were missing and that need to be intercepted